CHANGELOG.md

```diff
@@ -26,6 +26,16 @@ Changes by Version
     --collector.port
     ```
 
+* Normalize CLI flags to use host:port addresses ([#1827](https://github.com/jaegertracing/jaeger/pull/1827), [@annanay25](https://github.com/annanay25))
```

Comment:
Please make sure (with tests) that all new `host-port` flags also accept just the raw port number, without leading `:`, as it's the most common usage.